libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/oracle/OBOracleLessThan400SchemaAccessor.java

```diff
@@ -50,7 +50,7 @@
  * @Description: 适用于的 OB 版本：[2270, 400)
  */
 @Slf4j
-public class OBOracleLessThan400SchemaAccessor extends OBOracleSchemaAccessor {
+public class OBOracleLessThan400SchemaAccessor extends OBOracleBetween4000And4100SchemaAccessor {
```

Comment:
so, is there any differences between extending `OBOracleSchemaAccessor` and extending`OBOracleBetween4000And4100SchemaAccessor`?

Reply:
no, the only difference is the yaml file